mkdocs.yml

```diff
@@ -0,0 +1,8 @@
+# yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
+
+site_name: Operator Controller documentation
+theme:
+ name: "material"
+
+repo_url: https://github.com/operator-framework/operator-controller
+repo_name: github/operator-controller
```

Comment:
Based on the [MkDocs configuration documentation for `repo_name`](https://www.mkdocs.org/user-guide/configuration/#repo_name), it looks like this is determined by default by the `repo_url` and that `github/operator-controller` isn't a valid value for this and it should be one of `GitHub`, `Bitbucket`, `GitLab`

Reply:
Good catch, removed that line.